### PR TITLE
Fix arguments in QgisApp::about() for exported source code

### PR DESCRIPTION
**Pull request by agiudiceandrea**
## Description

Fixes arguments of QGIS code branch string and link in QgisApp::about() for exported source code.

### CODE REVIEW

File: src/app/qgisapp.cpp [hunk 1]
```diff
@@ -5111,8 +5111,15 @@ void QgisApp::about()
 
     if ( QString( Qgis::devVersion() ) == QLatin1String( "exported" ) )
     {
-      versionString += QStringLiteral( "%1</td><td><a href=\"https://github.com/qgis/QGIS/tree/release-%1_%2\">Release %1.%2</a></td>" )
-                       .arg( tr( "QGIS code branch" ) ).arg( Qgis::versionInt() / 10000 ).arg( Qgis::versionInt() / 100 % 100 );
+      if ( Qgis::version().endsWith( QLatin1String( "Master" ) ) )
+      {
+        versionString += QStringLiteral( "%1</td><td><a href=\"https://github.com/qgis/QGIS/tree/master\">master</a></td>" ).arg( tr( "QGIS code branch" ) );
```
Comment (rouault) [inline src/app/qgisapp.cpp:5116]: the code would be even clearer / simpler if versionString += tr( "QGIS code branch" ) was done before the if()

Comment (agiudiceandrea) [reply src/app/qgisapp.cpp]: You are right, thank you for pointing that out. Now it should be better, I think.

**Comment by agiudiceandrea** [comment]
The bug affects 3.10 and 3.12 branches, along with master branch. @rouault do you think it's worth backporting the fix?

**Comment by rouault** [comment]
hum if I've well understood your fix, it adds a special case for "master", so I don't see any benefits for backporting to branches, which by definition are not master. Or did I miss something ?

**Comment by agiudiceandrea** [comment]
The patch was originally intended to only fix the arguments in
```
QStringLiteral( "%1</td><td><a href=\"https://github.com/qgis/QGIS/tree/release-%1_%2\">Release %1.%2</a></td>" )
                       .arg( tr( "QGIS code branch" ) ).arg( Qgis::versionInt() / 10000 ).arg( Qgis::versionInt() / 100 % 100 );
```

as you can see, there are %1 and again %1 and %2, but tree arg(), creating the incorrect string
QGIS code branch | Release QGIS code branch.3
-- | --

with also incorrect link to the code branch on GitHub.

Fixing this, I also noticed that a special case for master branch was needed.

**Comment by rouault** [comment]
> there were %1 and again %1 and %2, but three arg(),

ah ok, missed that ! As I've already merged, I don't think the added labels will work. Can you issue manual backport pull requests ?

**Comment by rouault** [comment]
Nevermind: the backport bot did the job

**Pull request merged**